test/environments.js

```diff
@@ -49,7 +49,7 @@ function usingUnifiedTopology() {
  * @param {*} discoverResult
```

Comment:
Are these environment files still being used? They do contain important metadata, but they also still contain a lot of the orchestration code that we want to get rid of.

Reply:
Which environment files are you referring to? Do you mean the way environments are created here in different topologies?

Reply:
Sorry, i meant "Are the environments in this file still being used?" I'm asking:

a) are the environments still being added to the mocha test context
b) are the environments still setting up and tearing down servers using `mongodb-topology-manager`

Reply:
For now, we're still doing both a and b, but the plan is to get rid of `mongodb-topology-manager`.

Reply:
here's the PR for getting rid of `mongodb-topology-manager`: https://github.com/mongodb/node-mongodb-native/pull/2100